.github/workflows/ci.yml

```
@@ -151,8 +151,46 @@ jobs:
       - name: Run specs
         run: npm run js-api-spec -- --sassPackage ../dart-sass/build/npm --sassSassRepo language
 
+  js_api_dart_sass_browser:
+    name: "JS API | Dart Sass | Browser"
+    runs-on: ubuntu-latest
+    if: "github.event_name != 'pull_request' || !contains(github.event.pull_request.body, 'skip browser')"
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: browser-actions/setup-chrome@v1
+      - run: npm install
+      - uses: dart-lang/setup-dart@v1.2
+        with: {sdk: stable}
+      - uses: browser-actions/setup-chrome@v1
+
+      - name: Install Dart Sass
+        run: |
+          # TODO update this to use https://github.com/sass/dart-sass.git
+          git clone https://github.com/oddbird/dart-sass.git ../dart-sass --depth 1 --branch browser
```

Before merging, this should be updated:

```suggestion
          git clone https://github.com/sass/dart-sass.git ../dart-sass --depth 1
```